core/arch/arm/mm/core_mmu_lpae.c

```diff
@@ -425,6 +425,14 @@ void core_mmu_set_default_prtn(void)
 {
 	core_mmu_set_prtn(&default_partition);
 }
+
+void core_mmu_set_default_prtn_tbl(void)
+{
+	size_t n;
```

Comment:
n = 0;

Reply:
What is idea behind this? `n` gets initialized in the loop below. 

As far, as I know most coding standards require not to initialize variable during declaration if it is being initialized later.

Reply:
actually it does, bullet 3. in [coding standards](https://optee.readthedocs.io/en/latest/general/coding_standards.html).
> 3.  **All** variables **shall be** initialized ...

we should make a perm-link for this clause.
not ideal but

Reply:
sorry for the burden about this somewhat cosmetic concern. 
it helps source tree consistency to have consistent coding style(s). we better be fixed this now than with various fixup changes far later on.

Reply:
My bad, I will fix it.